lib/feature/global_app_widgets/global_app_bar.dart

```diff
@@ -0,0 +1,56 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_screenutil/flutter_screenutil.dart';
+import 'package:split/core/widgets/base_stateless_widget.dart';
+import 'package:split/res/app_colors.dart';
+
+// ignore: must_be_immutable
+class GlobalAppBar extends BaseStatelessWidget implements PreferredSizeWidget {
+  GlobalAppBar({required this.title, Key? key, required this.emptyGroupList})
+      : super(key: key);
+  final String title;
+  final bool emptyGroupList;
+  @override
+  Widget baseBuild(BuildContext context) {
+    return AppBar(
+      // foregroundColor: Colors.black,
+      backgroundColor: AppColors.appBarBackground,
+      shape: RoundedRectangleBorder(
+          borderRadius: BorderRadius.only(
+              bottomLeft: Radius.circular(10.r),
+              bottomRight: Radius.circular(10.r))),
+      actions: emptyGroupList == false
+          ? const [
```

Comment:
get action list from contractor
add name contractor to use in home and handle onpress

Reply:
I make it only for groups feature so, idon't need to do that